packages/terra-hookshot/tests/nightwatch/components/DefaultHookshot.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Hookshot from '../../../lib/Hookshot';
+
+class HookshotStandard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.state = { open: false };
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    const hookshotContent = (
```

Comment:
Can we update the styling on this so it presents a bit better. 
Current display when open:
<img width="142" alt="screen shot 2017-10-10 at 4 21 41 pm" src="https://user-images.githubusercontent.com/633148/31411440-46bebf40-add7-11e7-8a4e-822f272239a9.png">


I'm also fine with removing the default all together, the other tests cover everything I think this covers.

Reply:
What browser shows that? I'm not seeing it displayed that way.

Reply:
Chrome, it was [this url](https://terra-core-deployed-pr-879.herokuapp.com/static/#/tests/hookshot-tests/default) but it looks fixed now: 
<img width="214" alt="screen shot 2017-10-10 at 8 34 21 pm" src="https://user-images.githubusercontent.com/633148/31418224-6bc303fa-adfa-11e7-8669-db3bd30b6bf7.png">